web/index.php

```diff
@@ -58,17 +59,10 @@
     ->add('q', 'search')
     ->add('type', 'choice', array(
         'choices' => array(
-            'any'     => 'All packages',
             'plugin'  => 'Plugins',
             'theme'   => 'Themes',
         ),
     ))
-//    ->add('active_only', 'choice', array(
```

Comment:
I think I commented it in a previous commit and @louim removed it. Which is fine with me.